components/rhel/rhel-ose/Vagrantfile

```diff
@@ -47,10 +47,8 @@ Vagrant.configure(2) do |config|
     config.registration.password = ENV['SUB_PASSWORD']
   end
 
-  config.vm.provision "shell", inline: <<-SHELL
-    systemctl enable openshift
-    systemctl start openshift
-  SHELL
+  # vagrant-service-manager plugin will configure openshift service by default for CDK
+  # config.servicemanager.services = "docker, openshift"
```

Comment:
>Why not remove it completely? as this is not required.

This is to give user with some hints about default setup right in Vagrantfile. Since the later block is showing up the connection information (echo) , however there is not configuration of service. IMO, we should keep those lines as commented in the Vagrantfile.